lib/vssolution3d.cpp

```diff
@@ -2695,11 +2894,11 @@ void VisualizationSceneSolution3d::PrepareCuttingPlane2()
 
             if (nodes.Size() == 3)
             {
-               DrawTriangle(cplane_buf, p, c, minv, maxv);
+               DrawCutTriangle(cplane_buf, p, c, minv, maxv);
```

Comment:
Should these lines also check for `cut_lambda=0`?

Reply:
Thanks for catching that, I removed it.